cfgov/unprocessed/css/organisms/header.less

```diff
@@ -83,6 +83,10 @@
             & > .m-global-search {
                 float: right;
             }
+
+            > .m-global-search {
+                width: 340px;
```

Comment:
@anselmbradford I got this from the existing form width, it's necessary to use absolute positioning on the form.